src/views/Home/components/LostAndFoundCard/index.tsx

```diff
@@ -203,6 +229,21 @@ const LostAndFoundCard = () => {
             ) : null}
           </>
         )}
+        {foundReports && foundReports.length > 0 && (
```

Comment:
Yes that was intentional, i let only one of them show up not all of them. I can change that if you think all of them should show. But i think it made sense if only one of them showed up.

Reply:
Yeah, I think all of them should show up, unless you have a specific reason to only show one.  Especially since you have them linking directly to the edit page now, it would be good to be able to access and link to the whole list from there.  If you only show one, I would, as a user, expect it to then bring me to a list of all of them, so I can pick the one to look at.